docs/quickstarts/nextjs.mdx

```diff
@@ -120,7 +120,11 @@ description: Add authentication and user management to your Next.js app.
             <header className="flex justify-end items-center p-4 gap-4 h-16">
               <SignedOut>
                 <SignInButton />
-                <SignUpButton />
+                <SignUpButton>
+                  <button className="bg-[#6c47ff] font-medium rounded-md text-sm h-[1.875rem] px-3 ring-1 text-white ring-[#6c47ff] cursor-pointer">
```

Comment:
Do we need the ring usage here? Are you looking to make the button taller by 2px?

This usage creates a dead spot where I am hovering over the button edges, but its not clickable. I'd recommend dropping the ring and increasing the height.

---

Alternatively, I think we could more closely match the buttons Next.js uses in the homepage template like the following:


```tsx
<SignUpButton>
  <button className="bg-[#6c47ff] text-white rounded-full font-medium text-sm sm:text-base h-10 sm:h-12 px-4 sm:px-5">
    Sign Up
  </button>
</SignUpButton>
```

![Screenshot 2025-06-20 at 7 28 06 PM](https://github.com/user-attachments/assets/e7468dec-c580-4df5-a78e-a6768f977a2c)